Security/Firewall/OAuthListener.php

```diff
@@ -29,34 +30,47 @@
 class OAuthListener implements ListenerInterface
 {
     /**
-     * @var \Symfony\Component\Security\Core\SecurityContextInterface
+     * @var SecurityContextInterface
+     *
+     * @deprecated since 1.4.3, to be removed in 2.0
      */
     protected $securityContext;
 
     /**
-     * @var \Symfony\Component\Security\Core\Authentication\AuthenticationManagerInterface
+     * @var TokenStorageInterface|SecurityContextInterface
+     */
+    protected $tokenStorage;
```

Comment:
please make the new property private. Using private visibility makes maintenance much easier, as we don't need to care about BC on them